src/qt.ts

```diff
@@ -103,6 +107,9 @@ export class Qt {
     public async designerFilename(): Promise<string> {
         let searchdirs = [];
         let filesnames = ["designer" + tools.exeExtension(), "Designer" + tools.exeExtension()];
+        if(process.platform == "linux" && !await tools.fileExists("/etc/apt")) {
+            filesnames = ["designer-qt5" + tools.exeExtension(), "designer-qt6" + tools.exeExtension()];
```

Comment:
* Any special reason why you also check `/etc/apt`?
* I would suggest to extend the `filesnames` variable, so that it contains the orignal value + the 2 new variants
  Just for the rare case that `designer` will be just named `designer`